src/test/ui/consts/const-match-pattern-arm.stderr

```diff
@@ -7,6 +7,9 @@ LL | |     Some(value) => true,
 LL | |     _ => false
 LL | | };
    | |_^
+   |
```

Comment:
Can we gather all control flow tests into `consts/control-flow` so that it's better organized?

Reply:
I've done this for the obvious files, and will try to do this from now on.